google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/PublisherSnippets.java

```diff
@@ -54,24 +67,112 @@ public void onFailure(Throwable t) {
         System.out.println("failed to publish: " + t);
       }
     });
-    // [END publish]
     return messageIdFuture;
   }
 
   /** Example of creating a {@code Publisher}. */
   // [TARGET newBuilder(TopicName)]
   // [VARIABLE "my_project"]
   // [VARIABLE "my_topic"]
-  public static void newBuilder(String projectName, String topicName) throws Exception {
-    // [START newBuilder]
```

Comment:
it was not used in the cloudsite docs, i cleaned up : so we know its clearly not.